datadog_lambda/wrapper.py

```diff
@@ -56,10 +56,16 @@
 if profiling_env_var:
     from ddtrace.profiling import profiler
 
+llmobs_api_key = None
 llmobs_env_var = os.environ.get("DD_LLMOBS_ENABLED", "false").lower() in ("true", "1")
 if llmobs_env_var:
+    from datadog_lambda.api import init_api
+    from datadog import api
     from ddtrace.llmobs import LLMObs
 
+    init_api()
+    llmobs_api_key = api._api_key
```

Comment:
Do you only need the api key here, or are we actually needing to load all of `datadog`?

Reply:
we just need the api key, but from what I can tell this `init_api` function sets the api key on `api._api_key` (and imports `api` from `datadog` itself). is there a better way I could go about getting the api key?

Reply:
Yes, we should avoid importing `datadog` if at all possible. It's a big package and eats up a ton of time just to import it.

I suggest pulling out the code that finds and sets the api key into its own function. Then, instead of importing `init_api`, you can just import and call that single function. https://github.com/DataDog/datadog-lambda-python/blob/fc0beaae643d59a4745ae6d8da163ad39cdb5970/datadog_lambda/api.py#L57-L78

Reply:
that makes sense to me - let me extract that part out!

Reply:
attempted in https://github.com/DataDog/datadog-lambda-python/pull/572/commits/b2cc1ba781a39fa4ebfb6e29704d42d956107815 - let me know what you think! i hoisted the API key to a global so we could re-use it to save a little time. I do see the import calls to `datadog` completely gone now (which looked to be maybe 40-60ms).